policy/terraform/gcp/gce/deny_compute_firewall_unrestricted_test.rego

```diff
@@ -1,91 +1,59 @@
 package terraform_gcp
 
+import rego.v1
+
 import data.testing as t
 
-test_deny_compute_firewall_unrestricted {
-    input := {
-        "resource": {
-            "google_compute_firewall": {
-                "f1": {
-                    "name": "f1",
-                    "source_ranges": ["0.0.0.0/0"],
-                    "allow": {
-                        "ports":["22", "5432"]
-                    }
-                }
-            }
-        }
-    }
+test_deny_compute_firewall_unrestricted if {
+	inp := {"resource": {"google_compute_firewall": {"f1": {
```

Comment:
Not sure about this formatting, maybe this and others like it was ported to v1 syntax manually? I would format consistently or just at least we can leave a small comment on this PR for how the porting was done for posterity's sake

Reply:
> all files were preprocessed with `opa fmt --write --v0-v1`

Reply:
I'll also copy this comment in the PR description for increased visibility 👌